app/src/main/java/org/simple/clinic/patient/Patient.kt

```diff
@@ -258,11 +256,12 @@ data class Patient(
       return queryModelsToPatientProfiles(loadPatientQueryModelsWithSyncStatus(syncStatus))
     }
 
-    fun patientProfile(patientUuid: UUID): Observable<List<PatientProfile>> {
-      return loadPatientQueryModelsForPatientUuid(patientUuid)
-          .map { queryModelsToPatientProfiles(it) }
-          .toObservable()
-    }
+    @Transaction
+    @Query("""
+      SELECT * FROM Patient
+      WHERE uuid = :patientUuid
+    """)
+    abstract fun patientProfile(patientUuid: UUID): Observable<List<PatientProfile>>
```

Comment:
Couldn't we just return `Observable<PatientProfile?>` here instead of nullable? Reason I ask is, we are returning `PatientProfile?` for `patientProfileImmediate` in later commits. So I want to check if we need to keep it consistent across both the usages.

Reply:
Yeah, we could. I would prefer to make this change in another PR because this would require changing downstream consumers of this method which is not related to the change in this PR.